components/constellation/constellation.rs

```diff
@@ -2644,6 +2644,8 @@ where
             ipc::channel().expect("Failed to create IPC channel!");
         let (storage_ipc_sender, storage_ipc_receiver) =
             ipc::channel().expect("Failed to create IPC channel!");
+        let (webgl_threads_sender, webgl_threads_receiver) =
```

Comment:
If there is a clean way to only create those if `Some(webgl_threads)`, we should do this.

Reply:
only way I see is to use `Option<IpcSender<()>>` and create channel when `Some(webgl_threads)`

Reply:
Yes that is a good idea.